packages/utilities/psammead-locales/moment/.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  extends: [
+    "eslint:recommended",
+    "plugin:es5/no-es2015"
+  ],
+  plugins: ['es5'],
+  rules: {
+    'object-shorthand': 0,
```

Comment:
Howcome these specific rules? is `plugin:es5/no-es2015` not enough?

Reply:
Unfortunately yes